app/assets/javascripts/hyrax.js

```diff
@@ -37,11 +37,16 @@
 //= require webcomponentsjs/0.5.4/CustomElements.min
 //= require time-elements
 
+//= require action_cable
+
 //= require hyrax/monkey_patch_turbolinks
+//= require hyrax/fileupload
```

Comment:
Moved this before the `almond` require, because otherwise it is unhappy.

Reply:
Might be nice to comment which components need to be before and after almond

Reply:
I like the idea. In practice, I don't know what needs almond and I'm not sure how I'd even figure that out. Maybe someone who is more knowledgeable about this part of the code could toss a new commit in here in the future that more rationally orders these requires with appropriate comments?